unix-domain-socket/src/test/java/docs/javadsl/UnixDomainSocketTest.java

```diff
@@ -63,15 +63,18 @@ public void aUnixDomainSocketShouldReceiveWhatIsSent() throws Exception {
     // #binding
     java.nio.file.Path path = // ...
         // #binding
-        Files.createTempFile("aUnixDomainSocketShouldReceiveWhatIsSent1", ".sock");
+        Files.createTempDirectory("UnixDomainSocketSpec").resolve("sock1");
     path.toFile().deleteOnExit();
 
     // #binding
     final Source<IncomingConnection, CompletionStage<ServerBinding>> connections =
         UnixDomainSocket.get(system).bind(path);
     // #binding
 
+    final CompletableFuture<ByteString> received = new CompletableFuture<>();
+
     // #outgoingConnection
+    ByteString sendBytes = ByteString.fromString("Hello");
```

Comment:
(this line could be moved down to below the part that makes it into the docs)

Reply:
:+1: